python/lsst/cp/pipe/utils.py

```diff
@@ -336,13 +339,13 @@ def irlsFit(initialParams, dataX, dataY, function, weightsY=None, weightType='Ca
     ----------
```

Comment:
Three functions here don't have numpydoc-style docstrings: `parseCmdlineNumberString`, `countMaskedPixels`, and `checkExpLengthEqual`.